sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

```diff
@@ -565,9 +565,9 @@
  "metadata": {
   "celltoolbar": "Tags",
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
+   "display_name": "model-validation",
```

Comment:
Should we keep the kernel spec same as previous.

Reply:
reverted